### ISSUE TITLE
I use korean letter(utf-8) in query, but It causes error.

### ISSUE THREAD

**Issue by song9446** [will/crystal-pg#52]
test.cr
```
con = PG.connect("postgres://song@localhost/blog")
con.exec("SELECT * FROM BLOG WHERE NAME='test';")
  # => it works very well,

con = PG.connect("postgres://song@localhost/blog")
con.exec("SELECT * FROM BLOG WHERE NAME='테스트';") 
  # => end of file reached (IO::EOFError)
```

terminal
```
~$ psql BLOG
BLOG=# SELECT * FROM BLOG WHERE NAME='테스트';
```
  => it works correct too.

  

I use Postgresql 9.4.7
I set all table's encoding type utf-8.

**Comment by asterite** [comment]
There was a very similar issue recently: #51 and it's solved on master. Can you check if this happens to you if you use the master version of this shard?

**Comment by song9446** [comment]
Oh man, It works perfect! Thanks for your answer!

**Comment by will** [comment]
I released 0.9.0 so this fix is in a released version instead of just on master
